ibis/pandas/execution/generic.py

```diff
@@ -139,17 +150,33 @@ def execute_cast_series_date(op, data, type, **kwargs):
     raise TypeError("Don't know how to cast {} to {}".format(from_type, type))
 
 
+@execute_node.register(ops.SortKey, pd.Series, bool)
+def execute_sort_key_series_bool(op, data, ascending, **kwargs):
+    return data
+
+
 def call_numpy_ufunc(func, op, data, **kwargs):
     if data.dtype == np.dtype(np.object_):
         return data.apply(functools.partial(execute_node, op, **kwargs))
     return func(data)
 
 
+@execute_node.register(ops.Negate, scalar_types)
+def execute_obj_negate(op, data, **kwargs):
```

Comment:
this is unavailable for some dtypes, e.g. datetimes & object, do you test?

Reply:
No, thanks for pointing out. I'll remove those types from this dispatch and add a test.

Reply:
Fixed